wmn-data.json

```diff
@@ -5418,14 +5433,19 @@
         "cat" : "gaming"
        },
        {
-        "name" : "steemit",
-        "uri_check" : "https://steemit.com/@{account}",
-        "e_code" : 200,
-        "e_string" : "blog",
-        "m_string" : "Page Not Found - Steemit",
-        "m_code" : 301,
-        "known" : ["petlover", "zalat"],
-        "cat" : "social"
+        "name": "Steemit",
+        "uri_check" : "https://signup.steemit.com/api/check_username",
+        "uri_pretty": "https://steemit.com/@{account}",
+        "post_body" : "{\"username\":\"{account}\"}",
+        "e_code": 200,
+        "e_string": "\"error\":",

```

Comment:
Please consider changing this to something more specific like `error_api_username_used`

Reply:
It's the same situation as with Stripchat.